src/state/configsSelectors.ts

```diff
@@ -19,6 +19,9 @@ export const getAppColorMode = (state: RootState) => state.configs.appColorMode;
 
 export const getFeeTiers = (state: RootState) => state.configs.feeTiers?.toArray();
 
+export const getStatefulOrderEquityTiers = (state: RootState) =>
+  state.configs.equityTiers?.statefulOrderEquityTiers?.toArray();
```

Comment:
I've started wrapping these in real selectors since this isn't a stable reference (I think).